pkgs/applications/networking/hyprspace/default.nix

```diff
@@ -0,0 +1,27 @@
+{ iproute2mac ? null, lib, stdenv, buildGoModule, fetchFromGitHub }:
```

Comment:
why the null as the default?
Why not have just have the propagatedBuildInputs be optional on darwin, just like you have.

Reply:
It complains that it doesn't exist on linux.

Reply:
Ok things.
- the convention is to pass arguments in order they are used, so you can put iproute2mac in the end.
- you need to pass iproute2mac in the all-packages file. Since it's part of darwin it should be something like
`hyprspace = callPackage ../applications/networking/hyprspace { inherit (darwin) iproute2mac; };`
After that the ci should pass. If you look at the bottom of the pr, next to x86_64 it says no attempt. If you look at the details it gives you the reason it failed.